models/demos/t3000/falcon40b/tests/test_perplexity_falcon.py

```diff
@@ -263,7 +263,7 @@ def test_perplexity_huggingface(
         ("prefill", 1, 2048, "BFLOAT8_B-DRAM", 64, 6.55, 0.56, 0.80),
         ("decode", 32, 128, "BFLOAT8_B-SHARDED", 64, 13.91, 0.46, 0.71),
         ("decode", 32, 1024, "BFLOAT8_B-SHARDED", 64, 7.79, 0.54, 0.78),
-        ("decode", 32, 2048, "BFLOAT8_B-SHARDED", 64, 6.96, 0.55, 0.79),
+        ("decode", 32, 2048, "BFLOAT8_B-SHARDED", 64, 6.96, 0.55, 0.79),  # TODO: Hangs on CI
```

Comment:
Do we have a ticket for this issue? Ideally we should create one and then add the issue number in the comment.

Reply:
Agreed!

Reply:
Here is the issue: https://github.com/tenstorrent/tt-metal/issues/12320